lib/Elastica/Search.php

```diff
@@ -22,6 +22,7 @@ class Search
     const OPTION_SIZE = 'size';
     const OPTION_SCROLL = 'scroll';
     const OPTION_SCROLL_ID = 'scroll_id';
+    const OPTION_QUERY_CACHE = 'query_cache';
```

Comment:
To make this work, it must also be added to _validateOptions: https://github.com/ruflin/Elastica/blob/master/lib/Elastica/Search.php#L273 Best would be to add a small tests to make sure it works as expected.